field/generator/internal/templates/element/conv.go

```diff
@@ -83,7 +83,15 @@ func (z *{{.ElementName}}) Text(base int) string {
 	{{- end}}
 }
 
+// BigInt sets and return z as a *big.Int
+func (z {{.ElementName}}) BigInt(res *big.Int) *big.Int {
```

Comment:
Using mixed receivers (pointer for some functions and regular for others) is discouraged. Better to explicitly copy.
https://github.com/ConsenSys/gnark-crypto/issues/242